<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>531</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>281</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Signin</string>
      </property>
      <widget class="QLabel" name="label_username">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_password">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_username">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_login">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_status">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_status_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>140</y>
         <width>261</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>271</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Admin</string>
      </property>
      <widget class="QPushButton" name="pushButton_members">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Manage members</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_inventory">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>30</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Manage inventory</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_memberships">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Memberships</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_logout_admin">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>331</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>Manager</string>
      </property>
      <widget class="QPushButton" name="pushButton_sales">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Sales report (day)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_purchases">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Total purchases by number</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_qty">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Qty sold by item name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_rebates">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Rebates</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_expirations">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Membership expirations</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_item_lookup">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Item lookup</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_member_lookup">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>170</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Member lookup</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_convert">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>240</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Convert membership</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_logout_manager">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>300</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>341</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Admin: Manage Members</string>
      </property>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>361</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Admin: Manage Inventory</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_inventory">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>451</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Admin: Manage Memberships</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_memberships">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>411</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Manager: Sales Report</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_report">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>361</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>Manager: Total Purchases sorted by number</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_purchases">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9">
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>431</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Manager: Quantity sold by item name</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_qty">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_10">
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>401</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Manager: Rebates</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_rebates">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>230</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_11">
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>431</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Manager: Manage membership expirations</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_expirations">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_12">
     <widget class="QGroupBox" name="groupBox_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>431</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Manager: Item Lookup</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_item_lookup">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_13">
     <widget class="QGroupBox" name="groupBox_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>441</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Manager: Member Lookup</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_member_lookup">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_14">
     <widget class="QGroupBox" name="groupBox_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>441</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Manager: Convert Membership</string>
      </property>
      <widget class="QPushButton" name="pushButton_back_convert">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
